README.md

```diff
@@ -1,61 +1,143 @@
+
+    We recommend setting up Percona Everest on the [Amazon Elastic Kubernetes Service (EKS)](https://docs.percona.com/everest/quickstart-guide/eks.html) or [Google Kubernetes Engine (GKE)](https://docs.percona.com/everest/quickstart-guide/gke.html).
+
+2. Verify that you have access to the Kubernetes cluster that you want to use with Everest. By default, Everest uses the kubeconfig file available under `~/.kube/config`. 
+
+    To verify access to the Kubernetes cluster, run the following command:
+   
+    ```sh 
+    kubectl get nodes
+    ```
+
+## Install Percona Everest
+
+To install and provision Percona Everest to Kubernetes:
+
+
+1. Install the latest version of the Everest CLI by running the following commands:
+
+    **Linux and WSL**
+        
+    ```sh
+    curl -sSL -o everestctl-linux-amd64 https://github.com/percona/percona-everest-cli/releases/latest/download/everestctl-linux-amd64
+    sudo install -m 555 everestctl-linux-amd64 /usr/local/bin/everestctl
+    rm everestctl-linux-amd64
+    ```
+
+    **macOS (Apple Silicon)**
+
+    ```sh
+    curl -sSL -o everestctl-darwin-arm64 https://github.com/percona/percona-everest-cli/releases/latest/download/everestctl-darwin-arm64
+    sudo install -m 555 everestctl-darwin-arm64 /usr/local/bin/everestctl
+    rm everestctl-darwin-arm64
+    ```
+
+    **macOS (Intel CPU)**
+
+    ```sh
+    curl -sSL -o everestctl-darwin-amd64 https://github.com/percona/percona-everest-cli/releases/latest/download/everestctl-darwin-amd64
+    sudo install -m 555 everestctl-darwin-amd64 /usr/local/bin/everestctl
+    rm everestctl-darwin-amd64
+    ```
+
+2. Install Everest and provision the Kubernetes cluster using one of the following commands:
+
+    ```sh
+    everestctl install
+    ```
+
+    Enter the specific names for the namespaces you want Everest to manage, separating each name with a comma.
+
+    !!! note 
+        Make sure that you enter at least one namespace.
+
+    Alternatively, you can set multiple namepaces in the headless mode:
 
-## Creating Kubernetes cluster
+      ```sh
+      everestctl install --namespaces <namespace-name1>,<namespace-name2> --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --skip-wizard
+      ```
+    
+    Replace `<namespace-name>` with the desired name for your namespace.
 
-You must have a publicly accessible Kubernetes cluster to use Everest. EKS or GKE is recommended, as it may be difficult to make it work with local installations of Kubernetes such as minikube, kind, k3d, or similar products. Everest does not help with spinning up a Kubernetes cluster but assists with installing all the necessary components for Everest to run.
+    !!! note
+        Ensure that you copy the authorization token displayed on the terminal in this step. You will need this token to log in to the Percona Everest UI.    
```

Comment:
@rasika-chivate I don't think this is rendered correctly in GitHub, AFAIK this ain't standard markdown.

Reply:
The formatting is correct. Let me check why its not rendering properly.